.gitignore

```diff
@@ -3,6 +3,6 @@
 *.iml
 .env
 .DS_Store
-venv/
+venvt/
```

Comment:
Do not need change this)
If you name it another better to add new, because venv is default name

Reply:
@viktory-koroliova Not fixed

Reply:
I definitely see it as fixed. I added a new catalog venvt as you suggested
![image](https://user-images.githubusercontent.com/117012788/224735333-512be0f8-8315-4cc4-8d5b-a278b15ea5f6.png)

Reply:
remove venvt/ from .gitignore